libvast/test/synopsis.cpp

```diff
@@ -80,11 +79,14 @@ FIXTURE_SCOPE(synopsis_tests, fixtures::deterministic_actor_system)
 
 TEST(serialization) {
   factory<synopsis>::initialize();
+  // auto x = synopsis_ptr{};
+  // CHECK_EQUAL(roundtrip(x), x);
+
   CHECK_ROUNDTRIP(synopsis_ptr{});
-  CHECK_ROUNDTRIP_DEREF(
-    factory<synopsis>::make(legacy_bool_type{}, caf::settings{}));
-  CHECK_ROUNDTRIP_DEREF(
-    factory<synopsis>::make(legacy_time_type{}, caf::settings{}));
+  // TODO CHECK_ROUNDTRIP_DEREF(
+  // TODO factory<synopsis>::make(legacy_bool_type{}, caf::settings{}));
+  // TODO CHECK_ROUNDTRIP_DEREF(
+  // TODO factory<synopsis>::make(legacy_time_type{}, caf::settings{}));
```

Comment:
Why are these commented out?

Reply:
I have reenabled these checks. I was not sure that they work still, but they do.